topostats/thresholds.py

```diff
@@ -17,17 +18,12 @@ def threshold(image: np.array, method: str = 'otsu', **kwargs: dict) -> float:
     -------
     float
         Threshold of image using specified method.
-
-    Examples
-    --------
-    FIXME: Add docs.
-
     """
     thresholder = _get_threshold(method)
     return thresholder(image, **kwargs)
 
 
-def _get_threshold(method: str = 'otsu'):
+def _get_threshold(method: str = "otsu"):
```

Comment:
Just noticed there's no return type here. I guess functions aren't somehow massaged into being a "type".

Reply:
Thanks for picking this up, hadn't thought about this myself so [looked it up](https://stackoverflow.com/questions/37835179/how-can-i-specify-the-function-type-in-my-type-hints). The return type here should be `Callable`, duly added.